IntegrationTests/UITests/CommandsDialogs/FormBrowseTests.LeftPanel.ReorderNodes.cs

```diff
@@ -155,7 +164,12 @@ public void RepoObjectTree_moving_node_across_hidden_trees_skips_them()
                     repoObjectTree.SetTreeVisibleByIndex(2, true);
 
                     // Reset currNodes, should be back to 4
-                    currNodes = repoObjectTree.TreeView.Nodes;
+                    if (currNodes.Count != 4)
+                    {
+                        var nodes = currNodes.OfType<TreeNode>().Select(n => n.Text).Join(", ");
+                        Console.WriteLine($"{MethodBase.GetCurrentMethod().Name} {nameof(currNodes)}: {nodes}");
+                    }
```

Comment:
💡 Consider using an approach like [`AssertEx.EqualOrDiff`]() for this. It makes an assertion about the list, and produces a usable diff message if the assertion fails.

https://github.com/dotnet/roslyn/blob/cc3067485f0b489abc5467fb9ea15e33c131bec2/src/Test/Utilities/Portable/Assert/AssertEx.cs#L233-L273

Reply:
📝 The change to use `EqualOrDiff` would best be implemented as an independent change to simplify the work here.